docs/install/local/README.md

```diff
@@ -56,30 +56,49 @@ xcode-select --install
 1. Create a directory to be the home of your FlowForge install. For example: `/opt/flowforge`
 
    For Linux/MacOS:
-
     ```
     sudo mkdir /opt/flowforge
     sudo chown $USER /opt/flowforge
     ```
+   
+   For Windows:
+    ```console
+    mkdir c:\flowforge
+    ```
 
 2. Download the Installer zip file from https://github.com/flowforge/installer/releases
 
 3. Unzip the downloaded file into a temporary location and copy its contents to
    the FlowForge directory
 
-   ```
+   For Linux/MacOS:
+    ```
     cd /tmp/
     unzip flowforge-installer-x.y.z.zip
     cp -R flowforge-installer-x.y.z/* /opt/flowforge
     ```
 
+   For Windows:
+    ```console
+    cd c:\temp
+    tar -xf flowforge-installer-x.y.z.zip
+    xcopy /E /I flowforge-installer-x.y.z c:\flowforge
+    ```
+
 4. Run the installer and follow the prompts
 
+   For Linux/MacOS:
     ```
     cd /opt/flowforge
     ./install.sh
     ```
 
+   For Windows:
+    ```console
+    cd c:\flowforge
+    install.bat
+    ```
+
 #### Installing as a service
 
 On Linux, the installer will ask if you want to run FlowForge as a service.
```

Comment:
How does the user run as a service on Windows?

Reply:
You don't easily, you need extra things like pm2 or https://www.hardill.me.uk/wordpress/2014/05/30/running-node-red-as-a-windows-or-osx-service/.